<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonPause">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>35</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>|  |</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFastForward">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFastRewind">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>380</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>380</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBefore">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>|&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenFiles">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>380</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemove">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>380</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonBefore</tabstop>
  <tabstop>pushButtonFastRewind</tabstop>
  <tabstop>pushButtonPause</tabstop>
  <tabstop>pushButtonFastForward</tabstop>
  <tabstop>pushButtonNext</tabstop>
  <tabstop>pushButtonOpenFiles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
